spec/integration/supported_ssgs_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+describe 'SupportedSsgs API' do
+  fixtures :accounts
+
+  path "#{Settings.path_prefix}/#{Settings.app_name}/supported_ssgs" do
+    get 'List all supported SSGs' do
+      tags 'supported_ssg'
+      description 'List all supported SSGs mapped to RHEL minor version'
+      operationId 'ListSupportedSsgs'
+
+      content_types
+      auth_header
+
+      include_param
```

Comment:
This param is not used, but we accept it. I guess it's okay to keep it documented, despite this resource not having any relationships. What do you think? It's this `include` param from the JSON spec: https://jsonapi.org/format/#fetching-includes

Reply:
Oh, I thought the include is for the attributes and not related resources.

Reply:
I'll keep it. :slightly_smiling_face: